Library/TeamTalkPy/TeamTalk5.py

```diff
@@ -1634,10 +1652,6 @@ def acquireUserAudioBlock(self, uStreamTypes: StreamType, nUserID: int) -> POINT
     def releaseUserAudioBlock(self, lpAudioBlock: POINTER(AudioBlock)) -> bool:
         return _ReleaseUserAudioBlock(self._tt, lpAudioBlock)
 
-    def getMediaFileInfo(szMediaFilePath) -> MediaFileInfo:
-        mfi = MediaFileInfo()

```

Comment:
And this function should be fully restored.
Moreover, it is out of scope from this pull request to modify it in any manner.